Sources/SwiftGen/Commander/CustomCommands.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  CustomCommands.swift
+//  swiftgen
+//
+//  Created by Philippe Bernery on 15/01/2018.
+//  Copyright © 2018 Backelite. All rights reserved.
+//
+
+import Commander
+import PathKit
+import StencilSwiftKit
+import SwiftGenKit
+
+private let templatePathOption = Option<String>(
```

Comment:
Don't repeat yourself, make the `templatePathOption` in `ParserCLICommands.swift` internal instead of private, and maybe move it to a separate file.

Reply:
Indeed